upgrade/1.2/README.md

```diff
@@ -35,6 +35,10 @@ For more information about modifying `customizations.yaml` and tuning for specif
 
 ## Relevant troubleshooting links for upgrade-related issues
 
+- General Upgrade troubleshooting
+
+  If execution of the upgrade procedures fails, it is safe to just rerun the failed script. If a rerun still fails, wait for 10s and rerun it again. If the issue persists, then refer to below troubleshooting procedures.
```

Comment:
might be better to say "procedure fails" or "procedures fail" depending on the intended meaning